src/orion/algo/evolution_es.py

```diff
@@ -201,43 +211,47 @@ class BracketEVES(HyperbandBracket):
 
     """
 
-    def __init__(self, evolution_es, budgets, repetition_id):
-        super(BracketEVES, self).__init__(evolution_es, budgets, repetition_id)
-        self.eves = self.hyperband
+    def __init__(
+        self, evolution_es: EvolutionES, budgets: list[BudgetTuple], repetition_id: int
+    ):
+        super().__init__(evolution_es, budgets, repetition_id)
+        self.eves = evolution_es
         self.search_space_without_fidelity = []
-        self._candidates = {}
+        self._candidates: dict[int, list[Trial]] = {}
 
+        self.mutate_attr: dict = {}
         if evolution_es.mutate:
+            # TODO: This is inconsistent with the docstring of the constructor, which says it can be
```

Comment:
TODO :)

Reply:
Should it be a string? or do we want a dict with {"function": "<function_path>"} ?

Reply:
As you point out in the comment, it could be a string or a dict. We should first test whether it is a string and if so set `mutate_attr = {'function': evolution_es.mutate}`.

Reply:
Fixed in e15258e96c880198314ad8b3399080efb2993122